### ISSUE TITLE
Right-clicking on Tesseract kicks me from server

### ISSUE THREAD

**Issue by Egocitius** [EnderMediaDev/The-1.7.10-Pack#790]
Hi!

Whenever I right-click on a Tesseract to configure it, I get kicked from our private server over hamachi with an "internal server error".

I can craft it, place it and break it, but not right-click it.

Other players on the server stay unaffected.

I don't use optifine

I use Windows 8.1

http://pastebin.com/e48UJYpX
crash happens at 20:27:XX

Hope you can help me!

Awesome mod btw, really enjoying it!

**Comment by The-1-7-10-Pack** [comment]
It's not a crash, but I need the server latest log that includes the error.

**Comment by Egocitius** [comment]
http://pastebin.com/dS5Rc36z

There you go!

**Comment by The-1-7-10-Pack** [comment]
What version of the modpack do you use?
Do you have any additional mods?

**Comment by Egocitius** [comment]
I updated from 0.7.7b to 0.7.9e yesterday but I didn't have a tesseract before so I don't know if the same issue occurred in the older version as well.

What I can say though is, that since I updated the pack I experienced extreme lag spikes sometimes. They were going so far that it took the server a full minute to recognize me placing a block or accessing the applied energistics-terminal.
I think I will downgrade today to see if the problems persists!

I don't use any other mods.
